docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -152,11 +151,11 @@ By default, conda build creates packages for the version of
 Python installed in the root environment. To build packages for
 other versions of Python, you use the ``--python`` flag, followed
 by a version. For example, to explicitly build a version of the
-Pyinstrument package for Python 3.3, use:
+click package for Python 3.3, use:
 
 .. code-block:: bash
 
-    conda-build --python 3.3 pyinstrument
+    conda-build --python 3.3 click
```

Comment:
`conda build --python 2.7 click`

Reply:
I know we can keep the `conda build` but do I need to change the Python number to 2.7 or can it stay at 3.3?

Reply:
Perhaps better to make it just 3. That will age better. I think it'll work...

Reply:
The purpose of that argument is to make the package for a specific python version. That version is implicitly the same as the miniconda install that has conda build in it. Doing this overrides that implicit default.